index.html

```diff
@@ -0,0 +1,213 @@
+                    <a class="black-nav-link" target="_self" href="#">Contact</a>
+                </div>
+
+            </div>
+            <div class="invisible-black-menu">
+                <div class="invisible-logo">
+                    <img alt="logo" class="inv-logo" src="https://noknok.com/wp-content/uploads/2019/10/tnw-logo-white.png">
+                </div>
+                <div class="invisible-grey">
+                    <p>TNW Programs</p>
+                </div>
+
+            </div>
+            <div class="white-nav">
+                <div>
+                    <a class="white-nav-link" target="_self" href="#">LATEST</a>
+                    <a class="white-nav-link" target="_self" href="#">HARD FORK</a>
+                    <a class="white-nav-link" target="_self" href="#">PLUGGED</a>
+                    <a class="white-nav-link" target="_self" href="#">README</a>
+                    <a class="white-nav-link" target="_self" href="#">GROWTH QUARTERS</a>
+                    <a class="white-nav-link" target="_self" href="#">SHIFT</a>
+                    <a class="white-nav-link" target="_self" href="#">NEURAL</a>
+                </div>
+
+                <div class="search-div">
+                    <img class="search-icon" alt="search" src="resources/iconmonstr-magnifier-5-240.png">
+                </div>
+                <div class="hamburger-button">
+                    <img class="hb-icon" alt="hb-icon" src="resources/hamburger-menu.png">
+                </div>
+            </div>
+        </div>
+
+        <div class="yellow-nav">
+            <a href="#" class="yellow-nav-link">Calling young digital talent in the Netherlands - Be recognized in the T500→</a>
+        </div>
+    </nav>
+
+    <div class="highlights">
+        <div  class="left-hl">
+            <img alt="TNWlogo" src="https://noknok.com/wp-content/uploads/2019/10/tnw-logo-white.png" class="logo">
+            <p class="hl-time">Tuesday- January 26, 2021</p>
+            <h3 class="hl-title main-hl">Opal cuts off your apps' internet access to help you stop doomscrolling</h3>
+            <div class="hl-detail"><p>By Ivan Metha </p><p>&#x1F553; 1 hour ago </p><p><img alt="share" src="resources/iconmonstr-share-1-16.png"> 6</p></div>
+            
+        </div>
+
+        <div  class="topright-hl">
+             <h4 class="hl-title">Google unveils privacy-friendly AI cookie killer</h4>
+            <div class="hl-detail"><p>By Thomas Macauley </p><p>&#x1F553; 2 hours ago </p><p><img alt="share" src="resources/iconmonstr-share-1-16.png"> 4</p></div>
+        </div>
+
+        <div  class="bottomright-hl">
+            <h4 class="hl-title">This Mac app automatically detects your duplicate files</h4>
+            <div class="hl-detail"><p>By Ivan Metha </p><p>&#x1F553; 2 hours ago </p><p><img alt="share" src="resources/iconmonstr-share-1-16.png"> 5</p></div>
+        </div>
+    </div>
```

Comment:
- **Try to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)**

**For example, in this case, we can use the `<section>` and `<article>` tag for these and other articles.**

```
  <section class="highlights">
    <article class="left-hl">
      ...
    </article>
  </section>
```